rpcs3/Emu/Cell/lv2/sys_net.cpp

```diff
@@ -507,6 +511,18 @@ error_code sys_net_bnet_bind(ppu_thread& ppu, s32 s, vm::cptr<sys_net_sockaddr>
 
 	const auto sock = idm::check<lv2_socket>(s, [&](lv2_socket& sock) -> sys_net_error
 	{
+		if (sock.type == SYS_NET_SOCK_DGRAM_P2P)
+		{
+			const u16 daport = reinterpret_cast<const sys_net_sockaddr_in*>(addr.get_ptr())->sin_port;
```

Comment:
```suggestion
			const u16 daport = vm::_ptr<sys_net_sockaddr_in>(addr.addr())->sin_port;
```
thats what I meant, here and in similar cases.

Reply:
Tbh I'm fine with how it is now and I don't think it makes any functional difference?